### DISCUSSION TITLE
个人建议修改一下简介

### DISCUSSION THREAD

**Discussion by Z-fly** [lanyeeee/copymanga-downloader#8] [General]
![image](https://github.com/user-attachments/assets/dbeeb678-7fc7-4392-ab78-d8baa7e672c4)

这个是另一个项目作者在他自己项目里设置的限制，非拷贝官方限制。我参与过他那个项目，我觉得他用意是不想影响到官方。
我个人建议不用特意在简介强调这个。

**Comment by lanyeeee** [comment]
拷贝官方的API就是有每分钟15次的限制，自己试一下就知道了
自己写个脚本请求这个接口，试试短时间请求15次之后返回什么
```
https://api.mangacopy.com/api/v3/comic/buyaoqifuwochangjingtongxue/chapter2/6dccabfc-ba0b-11ec-971a-024352452ce0
```

**Comment by Connor888t** [comment]
大佬，拷贝更新v0.10.3后一直风控怎么解决？
